com.unity.netcode.gameobjects/Runtime/SceneManagement/ISceneManagerHandler.cs

```diff
@@ -11,7 +11,7 @@ namespace Unity.Netcode
     internal interface ISceneManagerHandler
     {
         // Generic action to call when a scene is finished loading/unloading
-        struct SceneEventAction
+        class SceneEventAction
```

Comment:
Just curious why this change was necessary. Not sure how often these are created so I don't know what the GC cost is, but changing to a class certainly adds more than 0 GC, right?

Reply:
Yeah.. that should have been a class.
The previous scene event progress was returning a false positive but the completed was never triggering.

Reply:
@ShadauxCat 
SceneEventAction no longer exists.